.github/actions/build-frontend/action.yml

```diff
@@ -46,10 +46,24 @@ runs:
         yarn build:staging
       shell: bash
 
+    - name: SonarCloud Scan
+      uses: sonarsource/sonarcloud-github-action@de2e56b42aa84d0b1c5b622644ac17e505c9a049
+      env:
+        GITHUB_TOKEN: ${{ inputs.GITHUB_TOKEN }}
+        SONAR_TOKEN: ${{ inputs.SONAR_TOKEN }}
+      with:
+        args: >
+          -Dsonar.cpd.exclusions=frontend-react/**/*.test.*
+          -Dsonar.javascript.lcov.reportPaths=frontend-react/coverage/lcov.info
+          -Dsonar.coverage.exclusions=frontend-react/**/__mocks__/**,frontend-react/**/mocks/**,frontend-react/**/*.test.*
+          -Dsonar.sources=frontend-react/src
+          -Dsonar.projectKey=CDCgov_prime-data-hub
+          -Dsonar.organization=cdcgov
+
     - name: Tar frontend files
       shell: bash
       working-directory: frontend-react
-      run: tar -czvf static-website-react.tar.gz build
+      run: tar -czf static-website-react.tar.gz build
```

Comment:
I know the 'v' makes the output really long, but it could have debug or forensic value.

Reply:
I haven't seen any errors on the tar step before so probably fine to drop the v

Reply:
I haven't seen any errors on the tar step before so probably fine to drop the v

Reply:
It's only to archive the results - it will still fail if there is a problem (and then we can debug it)